pkg/controller/kabaneroplatform/stack-controller.go

```diff
@@ -78,7 +78,7 @@ func reconcileStackController(ctx context.Context, k *kabanerov1alpha2.Kabanero,
 	// Create a RoleBinding in the tekton-pipelines namespace that will allow
 	// the stack controller to create triggerbinding and triggertemplate
 	// objects in the tekton-pipelines namespace.
-	templateCtx["name"] = "kabanero-" + k.GetNamespace() + "-trigger-rolebinding"
+	templateCtx["name"] = "kabanero-" + k.GetNamespace() + "-stack-trigger-rolebinding"
```

Comment:
I think a similar change is needed in `cleanupStackController()`.

Reply:
Hi Tim, yep it most definitely needs the update. Thanks for catching it. I made the update.
Need to go see what i did to my finder tool that did not find it when i searched the code.